core/src/main/java/com/nisovin/magicspells/variables/GlobalVariable.java

```diff
@@ -37,5 +37,14 @@ public double getValue(String player) {
 	public void reset(String player) {
 		value = defaultValue;
 	}
-	
+
+	@Override
+	public double getMaxValue() {
```

Comment:
What is the reason for this? Doesn't the definition of the max value exist within a superclass?

Reply:
It does. I didn't see a way to access either, so I made these methods. If I had made them `public`, would that have fixed the issue? Or is there a different solution to this?